### DISCUSSION TITLE
Local Control of Bulbs

### DISCUSSION THREAD

**Discussion by tggman** [SecKatie/ha-wyzeapi#314] [Q&A] [answered]
Local Control sounds awesome!
How do I use this feature / where do I specify the IP address ? Can someone show me a quick example of the service call in yaml?

**Comment by SecKatie** [comment, marked as answer]
It is automatic for the color bulbs

**Comment by tggman** [comment]
Thanks for the quick reply! Again, this is a great feature!
Note:  I was initially unable to control 3 out of my 8 color bulbs.  However, after several HA restarts (for unrelated issues), they "self-heeled" so all is well as of now. 

Again, thanks for all the time and effort you are putting into this project. It's appreciated.
